lib/provider.js

```diff
@@ -146,7 +170,11 @@ const init = (providerConfig) => {
   return {
     async upload(file) {
       try {
-        const fullFileName = generateUploadFileName(basePath, file);
+        const fullFileName = generateUploadFileName(
+          basePath,
+          file,
+          config.shortFilePaths === true
```

Comment:
It should be better to set variable outside.
Create a new variable `const shortFilePaths` at the beginning of the method with default value `false` or `config.shortFilePaths` ?